app/resources/api/v2/submission_resource.rb

```diff
@@ -2,53 +2,168 @@
     # of the JSON:API standard.
     class SubmissionResource < BaseResource
-      # Constants...
-
-      immutable
-
-      # model_name / model_hint if required
-
       default_includes :uuid_object, :sequencing_requests
 
-      # Associations:
-
+      ###
       # Attributes
+      ###
+
       # CAUTION:
       # See app/controllers/api/v2/submissions_controller.rb
       # for field filtering, otherwise newly added attributes
       # will not show by default.
-      attribute :uuid, readonly: true
-      attribute :name, write_once: true
-      attribute :state, readonly: true
+
+      # @!attribute [r] created_at
+      #   @return [DateTime] the date and time the {Submission} was created as an ISO8601 string.
       attribute :created_at, readonly: true
+
+      # @!attribute [r] updated_at
+      #   @return [DateTime] the date and time the {Submission} was last updated as an ISO8601 string.
       attribute :updated_at, readonly: true
-      attribute :used_tags, write_once: true
+
+      # @!attribute [rw] lanes_of_sequencing
+      #   @return [Integer] the number of lanes of sequencing requested in the Submission.
+      #      This can only be written once on creation.
       attribute :lanes_of_sequencing, write_once: true
 
+      def lanes_of_sequencing
+        _model.sequencing_requests.size
+      end
+
+      # @!attribute [rw] name
+      #   @return [String] the name of the {Submission}.
+      #      This can only be written once on creation.
+      attribute :name, write_once: true
+
+      # @!attribute [w] order_uuids
+      #   This is declared for convenience where the {Order} resources are not available to set as a relationship.
+      #   Setting this attribute alongside the `orders` relationship will prefer the relationship value.
+      #   @deprecated Use the `orders` relationship instead.
+      #   @param value [Array<String>] the UUID of the {Order} resources associated with this {Submission}.
+      #   @return [Void]
+      #   @see #orders
+      attribute :order_uuids, writeonly: true
+
+      def order_uuids=(value)
+        @model.orders = value.map { |uuid| Order.with_uuid(uuid).first }
+      end
+
+      # @!attribute [r] state
+      #   @return [String] a string version of the state name for this {Submission}.
+      #     This is one of `building`, `pending`, `processing`, `ready`, `failed` or `cancelled`.
+      attribute :state, readonly: true
+
+      # @!attribute [rw] used_tags
+      #   @return [String] the tags that were used in this {Submission}.
+      #      This can only be written once on creation.
+      attribute :used_tags, write_once: true
+
+      # @!attribute [w] user_uuid
+      #   This is declared for convenience where the {User} is not available to set as a relationship.
+      #   Setting this attribute alongside the `user` relationship will prefer the relationship value.
+      #   @deprecated Use the `user` relationship instead.
```

Comment:
I could not understand the doc.
whatever is set last wins. user directly, or indirectly by setting the uuid, which finds the user and sets it. There is no preference.

Reply:
See above